docs/learn/learn-comparison-ethereum-2.md

```diff
@@ -195,51 +189,59 @@ parachain in the network.
 ### Interoperability and Message Passing
 
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} uses
-[Cross-Consensus Message Passing Format (XCM)](./learn-xcm.md) for parachains to send arbitrary
-messages to each other. Parachains open connections with each other and can send messages via their
-established channels. Given that [collators](./learn-collator.md) will need to be full nodes of the
-relay chain as well, they will be connected and can relay messages from parachain A to parachain B.
+[Cross-Consensus Messaging (XCM)](./learn-xcm.md) for parachains to send arbitrary messages to each
+other. Parachains open connections with each other and can send messages via their established
+channels. Given that [collators](./learn-collator.md) will need to be full nodes of the relay chain
+as well, they will be connected and can relay messages from parachain A to parachain B.
 
 Messages do not pass through the relay chain. Only validity proofs and channel operations do (open,
 close, etc.). This enhances scalability by keeping data on the edges of the system.
 
+Currently, Ethereum rollups can communicate using
+[shared sequencers](https://medium.com/@richardchen_81235/intro-to-shared-sequencing-1622d1fd51c9),
+which provide a common grounds of interoperability between layer two solutions.
+
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} will add a protocol called
-[SPREE](learn-spree.md) that provides shared logic for cross-chain messages. Messages sent with
+[SPREE](learn-spree.md) that provides shared logic for cross-consensus messages. Messages sent with
```

Comment:
Maybe here we can just mention Accords and then link to the page https://wiki.polkadot.network/docs/polkadot-direction#xcm-and-accords that also mentions SPREE.